### ISSUE TITLE
Specify extra environment variables for job template

### ISSUE THREAD

**Issue by dagstuan** [ansible/awx#5060]
##### ISSUE TYPE
 - Feature Idea/Question

##### SUMMARY
Is there any way to currently specify extra environment variables for a job template? I can specify extra environment variables for all jobs under `Settings -> Jobs`, but I haven't been able to find any way of doing this only for a single job or job template. More specifically I would like to specify the `display_skipped_hosts`-setting for a single job template without affecting the rest of the jobs.

**Comment by wenottingham** [comment]
No, there is only the global setting at the moment.

**Comment by bar0n36** [comment]
This would be very handy in some use cases I have with vast amounts of hosts and data where it would be good to exclude skipped hosts per Template, rather than globally (other smaller inventories, and things under testing I don't want to exclude it from or it will be confusing)

**Comment by LucianoRed** [comment]
I agree. I have just need it now. I am using a playbook for demo (read only) and the playbook didn't had the region on AWS to deploy. It would be much easier to place the environment variable on the job template or on the execution environment (prefer job template).
